### ISSUE TITLE
CSS minifier: pct are off

### ISSUE THREAD

**Issue by mikegleasonjr** [tdewolff/minify#100]
**Test case:**

```
.et_pb_gutters3.et_pb_row .et_pb_column_1_3 {
    width: 29.666%;
}
```

**EDIT:** with `Decimals` property to `0`

- on branch 2.0.0 (OK):

```
.et_pb_gutters3.et_pb_row .et_pb_column_1_3{width:29.666%}
```

- on branch master (**BUG**, **29.666%** became **3%**):

```
.et_pb_gutters3.et_pb_row .et_pb_column_1_3{width:3%}
```

**Comment by tdewolff** [comment]
I thought these was the result of rounding to eliminate decimal digits, but this seems to be a more thorough bug. What did you use for the number of decimals in the CSS options? The default (ie. non-rounding)?

Thanks for reporting, I'll get on this ASAP!

**Comment by mikegleasonjr** [comment]
I am not sure where those options are, I just use:

```golang
[...]
cssMinifier := &css.Minifier{}
min.Add("text/css", cssMinifier)
[...]
```

**Comment by tdewolff** [comment]
The default for `Decimal` is `-1` which means no rounding. `0` means rounding to zero decimals. So I assume you had the latter?

**Comment by mikegleasonjr** [comment]
Yes I instantiated the Minify object myself and I didn't pay attention about the `Decimals` property on master, so yes it was set to `0`. I closed #98, but here 33.33% with 0 decimals should be 33%.

**Comment by mikegleasonjr** [comment]
Thanks, as always, super fast
